cmd/kube-scheduler/app/options/options.go

```diff
@@ -197,37 +197,37 @@ func (o *Options) Validate() []error {
 
 // Config return a scheduler config object
 func (o *Options) Config() (*schedulerappconfig.Config, error) {
+	c := &schedulerappconfig.Config{}
+	if err := o.ApplyTo(c); err != nil {
```

Comment:
we apply the options to the config *first*, then use the `c.ComponentConfig....` for the rest of the function